bindings/python/src/OpenSpaceToolkitAstrodynamicsPy/EventCondition.cpp

```diff
@@ -58,11 +58,22 @@ inline void OpenSpaceToolkitAstrodynamicsPy_EventCondition(pybind11::module& aMo
 
         eventCondition_class
 
-            .def(init<const String&, const EventCondition::Criteria&>(), arg("name"), arg("criteria"))
+            .def(
+                init<const String&, const EventCondition::Criteria&, const Real&>(),
+                arg("name"),
+                arg("criteria"),
+                arg("target")
+            )
 
             .def("__str__", &(shiftToString<EventCondition>))
             .def("__repr__", &(shiftToString<EventCondition>))
 
+            .def("get_name", &EventCondition::getName)
```

Comment:
Not 100% about this MR but would be nice to have some kind of glossary (maybe as part of the doc strings) that say what all of these words mean. 

It's not very obvious what `target` or `criteria` means or what the difference is between `compute` and `evaluate`. Maybe adding a class-level docstring to Event conditions could help?

Reply:
Discussed this with Kyle and we agreed that the doxygen comments are sufficient for now...